src/js/shield_text.js

```diff
@@ -243,15 +213,17 @@ export function drawBannerText(ctx, text, bannerIndex) {
   ctx.fillStyle = "black";
 
   ctx.font = Gfx.shieldFont(textLayout.fontPx);
-  ctx.textBaseline = "top";
+  ctx.textBaseline = "alphabetic";
```

Comment:
The top pixel or so of each banner is cut off:

[<img src="https://user-images.githubusercontent.com/1231218/227705983-d790eef5-1079-4d7c-9d55-5ebda3e48bb6.png" width="400" alt="Downingtown">](https://zelonewolf.github.io/openstreetmap-americana/#map=15/40.00801/-75.6983)